docs/reference/README.md

```diff
@@ -1,7 +1,3 @@
 # Reference
 
-- [Glossary](/reference/glossary.md): Terminology and concepts that are important or useful to know when working with ApostropheCMS
```

Comment:
We don't have a good plan to maintain this. I think we could create a proper landing page when the reference is more mature and we have a more complete structure.

Reply:
I guess I wasn't clear in my note. the structure of the reference section is evolving. Rather than continually updating this page and adding relatively obvious descriptors I'm instead redirecting to the glossary page and letting the sidebar navigation do the work.

Reply:
✅